examples/asia

```diff
@@ -0,0 +1,3651 @@
+# tzdb data for Asia and environs
```

Comment:
Could this make use of the file in the `tz` submodule instead?

Reply:
That's a separate issue from merging the repos, let's do it in follow-up!